mlflow/utils/validation.py

```diff
@@ -356,6 +361,21 @@ def _validate_model_version(model_version):
         )
 
 
+def _validate_model_alias_name(model_alias_name):
```

Comment:
Is there a unit test for this?  If not, can we add one to test the regex pattern?

Reply:
Great call! Added the tests